zebra-chain/src/orchard/tests.rs

```diff
@@ -1,3 +1,4 @@
 mod preallocate;
 mod prop;
-pub mod test_vectors;
+mod test_vectors;
+mod vectors;
```

Comment:
Since these are specifically about the note commitment trees, maybe this module of tests can be named as such, to avoid confusion with test_vectors.rs

Reply:
Renamed in 97ec26174